src/injected/visualization/root-container-creator.ts

```diff
@@ -9,6 +9,6 @@ export class RootContainerCreator {
         this.htmlElementUtils.deleteAllElements(`#${id}`);
         const root = document.createElement('div');
         root.id = id;
-        this.htmlElementUtils.querySelector('body').appendChild(root);
+        this.htmlElementUtils.getBody().appendChild(root);
```

Comment:
The intention is for this to do the right thing on a page that uses `<frameset>` instead of `<body>`, right? I don't think making it work is quite this simple; MDN doesn't document `<frameset>` as supporting any child that isn't a `<frame>` element, and in local testing on lastest stable new edge, it doesn't look like `<frameset><div>test</div></frameset>` renders the `<div>`

Reply:
(same comment on most of the other usages that involve mutating/adding children to `document.body`)

Reply:
This doesn't fully "fix" the frameset scenario; it's just a safer/better way of doing what we already did that also happens to not completely crash/break the app in the frameset scenario. As in, this would still not be ideal in the frameset scenario but it won't completely break (i.e. it'll finish scanning and show results).

I think the "right thing" here likely requires some UX work and I figured this was an improvement/mitigation of some sorts.

Reply:
I appreciate wanting to break up the full solution into parts, but I think it would be better to work out how to get the scanner to actually scan across `<frame>` boundaries before making the change to avoid the scanner crashing; I think having the scanner obviously not work is vastly superior to having it *look like* it works but actually not be scanning the content of `<frame>`s and giving false negatives.

Reply:
Per offline conversation, my bad, I misremembered how we interacted with this the last time this issue came up and I tried making this PR myself; it looks like we don't have the false negatives after all.